website/content/en/status/freshports.adoc

```diff
@@ -0,0 +1,44 @@
+=== Status Report Sample - This Will Become The Title
+
+Links: +
+link:https://freshports..org[FreshPorts] URL: link:freshports.org[] +
+link:https://news.freshports.org/[FreshPorts blog] URL: link:https://news.freshports.org/[]
+
+Contact: Dan Langille <dvl@FreeBSD,org>
+
+FreshPorts and FreshSource have reported upon FreeBSD commits for 20 years. They cover all commits, not just ports.
+
+FreshPorts tracks the commits and extracts data from the port Makefiles to create a database of information useful to both port maintainers and port users.
+
+For example, https://www.freshports.org/security/acme.sh/ shows the history of the <i>acme.sh</i> port, back to its creation in May 2017.
+Also available are dependencies, flavors, configuration options, and available packages.
+All of this is useful for both users and developers of ports.
+
+SQL Injection Attack
+
+In March, an SQL injection attack was noticed and the website was patched.
+Notices were sent out via our Twitter account, our status page, and a notice on the top of each page of the website.
+The immediate attack vector was shutdown and soon patched.
+Additional prevevntative patches were added across the website.
+Everything we know about has been fixed.
+Users were notified and advised to change their passwords.
+
+Details at:
+
+* https://news.freshports.org/2023/03/24/sql-inejection-issues-fixed/
+* https://news.freshports.org/2023/03/24/freshsource-code-fixes/
+
+Help Needed
+
+It has been over 22 years since FreshPorts stared. Others must take over eventually. I’d like to start that process now. There are several aspects to FreshPorts:
```

Comment:
Typo: `stared`.
While here, it would be nice for the status team if you could break this line in multiple lines, so that we have one sentence per line.

Reply:
That's not what I'm seeing in my copy. WTF?

```
It has been over 22 years since FreshPorts started.
Others must take over eventually.
I’d like to start that process now.
There are several aspects to FreshPorts:
```

Reply:
Yep, https://github.com/freebsd/freebsd-doc/compare/main...dlangille:freebsd-doc:main looks right.

Reply:
I confirm that it looks right in here too. What still looks wrong is the `Help needed` line instead of `==== Help needed`. If you want I can fix it for you at commit time.